lib/metadata/models/object/ObjectMD.js

```diff
@@ -595,7 +595,7 @@ module.exports = class ObjectMD {
      * @param {object} headers - Headers
      * @return {ObjectMD} itself
      */
-    setHeaders(headers) {
+    overrideMetadataValues(headers) {
```

Comment:
please update description of function above to: overrideMetadataValues (used for complete MPU and object copy)